Makefile

```diff
@@ -79,8 +81,29 @@ dev-setup:
 	@echo "--> Local Setup completed"
 	@echo "------------------"
 
+run-local:
+	@LOCAL_GOOS=$(LOCAL_GOOS) LOCAL_GOARCH=$(LOCAL_GOARCH) docker-compose -f \
+	$(STANDALONE_DIRECTORY)/docker-compose-core.yaml -f $(STANDALONE_DIRECTORY)/docker-compose-local.yaml \
+	up --build -d
+
+down-local:
+	@docker-compose -f \
+	$(STANDALONE_DIRECTORY)/docker-compose-core.yaml -f $(STANDALONE_DIRECTORY)/docker-compose-local.yaml \
+	down -v
+
+run-prod:
+	@docker-compose -f \
+	$(STANDALONE_DIRECTORY)/docker-compose-core.yaml -f $(STANDALONE_DIRECTORY)/docker-compose-prod.yaml \
+	up --build -d
+
+down-prod:
+	@docker-compose -f \
+	$(STANDALONE_DIRECTORY)/docker-compose-core.yaml -f $(STANDALONE_DIRECTORY)/docker-compose-prod.yaml \
+	down -v
+
 run-x86:
-	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.yaml up -d
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose-core.yaml \
+	-f $(STANDALONE_DIRECTORY)/docker-compose-prod.yaml up -d
```

Comment:
I suppose we can revert these changes, since `prod` and `local` targets are suffice.

Reply:
@prashant-shahi Oh, you mean removing `run-x86` / `down-x86`? I can do that - but that'd mean existing workflows might break (e.g. if people are already relying on using these steps to run SigNoz on their clusters).

Reply:
There is also documentation changes to CONTRIBUTING.md which are going to be huge. Probably going to do that after this is merged.

Reply:
Yeah that makes sense. Updated. Let me know if it makes sense.

Reply:
Looks good :+1: